src/plugins/vis_types/xy/tsconfig.json

```diff
@@ -9,7 +9,8 @@
   "include": [
     "common/**/*",
     "public/**/*",
-    "server/**/*"
+    "server/**/*",
+    "*.ts"
```

Comment:
It's because of the top-level `vis_types/xy/config.ts` which sits outside of `server` and `public` (I copied the structure they have used in the other `vis_types` plugins)

Reply:
Of course, this could be removed by relocating the config to the server directory, just wasn't sure how much the app team cares about the existing convention they're using.

Reply:
Follow the structure of the others vis seems good